ext/afform/core/afform.php

```diff
@@ -127,6 +127,33 @@ function afform_civicrm_upgrade($op, CRM_Queue_Queue $queue = NULL) {
  */
 function afform_civicrm_managed(&$entities) {
   _afform_civix_civicrm_managed($entities);
+
+  $afforms = \Civi\Api4\Afform::get(FALSE)
+    ->setSelect(['title', 'name', 'module_name', 'directive_name'])
+    ->addWhere('is_dashlet', '=', TRUE)
+    ->execute();
+  $afforms = $afforms->getArrayCopy();
+
+  foreach ($afforms as $afform) {
+    $entities[] = [
+      'module' => E::LONG_NAME,
+      'name' => 'dashlet_' . $afform['name'],
```

Comment:
Maybe to avoid confusion:

```suggestion
      'name' => 'afform_dashlet_' . $afform['name'],
```

Reply:
I merged this because it seemed innocuous - then remembered that managed entities are namespaced (identified by tuple `module,name`), so the only confusion is among us humans. :)